docs/table_design/Data_distribution.md

```diff
@@ -1,6 +1,10 @@
 # Data distribution
 
-When you create a table, you must specify the data distribution method by configuring the number of partitions and the number of tablets in a table. An appropriate data distribution method helps evenly distribute data across the nodes of your StarRocks cluster, reduces table scans, makes full use of the concurrency of the cluster, thereby improving query performance.
+When you create a table, you must specify the data distribution method by configuring the partitioning and bucketing strategy in a table. An appropriate data distribution method helps evenly distribute data across the nodes of your StarRocks cluster, reduces table scans, makes full use of the concurrency of the cluster, thereby improving query performance.
+
+> NOTICE
+>
+> Since v2.5.7, StarRocks can set the number of buckets (BUCKETS) automatically when a table is created or a partition is added. You no longer need to manually set the number of buckets. However, if the performance does not meet your expectations after StarRocks sets the number of buckets automatically and you are familiar with the bucketing mechanism, you can still [manually set the number of buckets](#determine-the-number-of-buckets).
```

Comment:
However, if the performance does not meet your expectations after StarRocks automatically sets the number of buckets xxx